src/main/java/org/kitteh/irc/client/library/util/NagIgnorer.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * * Copyright (C) 2013-2019 Matt Baxter https://kitteh.org
+ *
+ * Permission is hereby granted, free of charge, to any person
+ * obtaining a copy of this software and associated documentation
+ * files (the "Software"), to deal in the Software without
+ * restriction, including without limitation the rights to use, copy,
+ * modify, merge, publish, distribute, sublicense, and/or sell copies
+ * of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+ * BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ * ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.kitteh.irc.client.library.util;
+
+import org.checkerframework.checker.nullness.qual.NonNull;
+import org.kitteh.irc.client.library.exception.KittehNagException;
+
+import java.util.function.Consumer;
+
+public class NagIgnorer {
```

Comment:
Needs to include documentation on application and use, please, to provide context.

Reply:
Yep yep. Just throwing out a possible idea. Probably would throw it into a sub package in util too, to not clutter too much, and maybe gain some friends of other little goodies. Or a separate class. I dunno.